euphonic/readers/castep.py

```diff
@@ -1,29 +1,46 @@
 import re
 import struct
+from typing import Dict, Optional, Any
 
 import numpy as np
 
 from euphonic import ureg
 from euphonic.util import is_gamma
 
 
-def _read_phonon_data(filename, cell_vectors_unit='angstrom',
-                      atom_mass_unit='amu', frequencies_unit='meV'):
+def _read_phonon_data(filename: str,
```

Comment:
This seems public, maybe drop the underscore? The name is a bit generic but in the context of `euphonic.readers.castep` it's fairly clear that we're reading CASTEP files so I guess that's fine.